operator/storage.go

```diff
@@ -40,6 +40,10 @@ func (s *storage) SaveSyncOffset(offset *eth1.SyncOffset) error {
 	return s.db.Set(prefix, syncOffsetKey, offset.Bytes())
 }
 
+func (s *storage) cleanSyncOffset() error {
```

Comment:
@amir-blox cleaning the offset doesn't clean the actual shares right? doesn't it mean the node could potentially still use old non deleted shares?

Reply:
Theoretically yes as the sync offset and shares uses different `collection` (shares cleanup is on validator storage), 
but it is the same underlying DB and therefore it could happen if the DB returns error for one operation and success for the other.

Reply:
@amir-blox maybe delete offset and shares?

Reply:
I can also fail to start if we can't delete something